test/fixtures/templates/good/resource_properties.yaml

```diff
@@ -218,6 +218,10 @@ Outputs:
           - ElasticLoadBalancer
           - DNSName
 Parameters:
+  azList:
+    Type: "AWS::SSM::Parameter::Value<List>"
```

Comment:
I just added a note on the issue, GH didn't render what i was trying to show properly. 
"AWS::SSM::Parameter::Value\<List\<String\>\>"

is the correct syntax here.

Reply:
Thanks.